### PR TITLE
Creating test for math-magicians

### PR DESCRIPTION
**Pull request by igorkol91**
In this milestone we created tests for:
- unit tests for components
- tests for user interaction with the nav links
- tests for user click on the calc buttons 
- tests for the calculator logic

### CODE REVIEW

File: src/tests/components/Navbar.test.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    expect(screen.queryByText(/Mathematics is not about numbers/)).toBeInTheDocument();
+  });
+
+  test('that user to redirected to the Calculator page on clicking Calculator', () => {
+    render(<Router><Navbar /></Router>);
+    expect(screen.queryByText('Lets do Some Math:')).toBeNull();
+    userEvent.click(screen.getByText('Calculator'));
+    expect(screen.queryByText('Lets do Some Math:')).toBeInTheDocument();
+  });
+});
```
Comment (AshakaE) [inline src/tests/components/Navbar.test.js]: - [ ] Kindly review the navbar test, it's failing... see below

image

**Review by AshakaE — CHANGES_REQUESTED** [review]
### Hi Team 🙋‍♀️, 

Good job on the pull request! ✔️ 
README is 👌 
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


#  Required Changes ♻️ 🚧
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by george-shammar — APPROVED** [review]
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

:heavy_check_mark: Awesome!! All 36 test cases are passing. :clap: 

Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by enkog** [comment]
Hello @george-shammar 👋 

Thank you for reviewing our project.

Cheers 🍷

**Pull request merged**